charts/descheduler/values.yaml

```diff
@@ -8,7 +8,7 @@ kind: CronJob
 image:
   repository: k8s.gcr.io/descheduler/descheduler
   # Overrides the image tag whose default is the chart version
-  tag: ""
+  tag: "v20220127-v0.22.0-79-gafb1d75ce"
```

Comment:
Just wondering if we explicitly need to mention tag value in values.yaml. I think this should come from the version in charts.yaml ? 🤔

Reply:
yet it should use version in charst.yaml, but since the release 1.23 not including the pr: https://github.com/kubernetes-sigs/descheduler/pull/731, so we need to add the image tag which build via https://github.com/kubernetes-sigs/descheduler/pull/731 merge

Reply:
But I think the image version should automatically be pointing to underlying tag. So just thinking why we need to hard code it here. Some context I've been trying to follow here: https://github.com/kubernetes-sigs/descheduler/pull/734#issuecomment-1049051241 Do we have any error log which might be coming when the helm chart is being deployed?

Reply:
for the release 1.23 not including the pr #731 , so we can not sue the image auto build will not cover this part.
please correct me if i am wrong

Reply:
I don't think this is the fix for https://github.com/kubernetes-sigs/descheduler/issues/736, @JaneLiuL there was some more discussion about it in https://github.com/kubernetes-sigs/descheduler/pull/734 that might explain the problem better

For #731, that will be made available when we publish a patch release (v0.23.1)

Reply:
get, thanks for clarify, i will close the pr now :)